kerykeion/charts/charts_utils.py

```
@@ -894,18 +896,24 @@ def convert_decimal_to_degree_string(dec: float, format_type: Literal["1", "2",
     # Ensure the input is a float
     dec = float(dec)
 
-    # Calculate degrees, minutes, and seconds
-    degrees = int(dec)
-    minutes = int((dec - degrees) * 60)
-    seconds = int(round((dec - degrees - minutes / 60) * 3600))
-
-    # Format the output based on the specified type
+    # All three formats floor (toward −∞) the displayed unit via math.floor/divmod.
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)